packages/extractor/src/internal/Spotify.ts

```diff
@@ -283,7 +319,7 @@ export interface SpotifyTrack {
         }[];
     };
     artists: {
-        id: string;
+        uri: string;
```

Comment:
would not this break existing code that depend on this type? I think it is better to revert this? 🤔

Reply:
Although the interface is exported, I could not see it used anywhere yet, only in internal Spotify.ts

The id field for an artist is not even used in internal Spotify.ts as far as I could see, only the name property.

The actual response from Spotify API looks like this:
```
[
  {
    external_urls: {
      spotify: 'https://open.spotify.com/artist/5Matrg5du62bXwer29cU5T'
    },
    href: 'https://api.spotify.com/v1/artists/5Matrg5du62bXwer29cU5T',
    id: '5Matrg5du62bXwer29cU5T',
    name: 'Benny The Butcher',
    type: 'artist',
    uri: 'spotify:artist:5Matrg5du62bXwer29cU5T'
  }
]
```

Since this SpotifyTrack is like a DTO for Spotify track response, I think it's correct to use uri.

Without it, I will have to create a new interface for SpotifyTrack to return this:
```
artists: m.artists.map((m) => {
    return {
        name: m.name,
        uri: m.uri
    };
}),
```
Because m.uri is infact correct, that's the response from Spotify API.

Reply:
Although the interface is exported, I could not see it used anywhere yet, only in internal Spotify.ts

The id field for an artist is not even used in internal Spotify.ts as far as I could see, only the name property.

The actual response from Spotify API looks like this:
```
[
  {
    external_urls: {
      spotify: 'https://open.spotify.com/artist/5Matrg5du62bXwer29cU5T'
    },
    href: 'https://api.spotify.com/v1/artists/5Matrg5du62bXwer29cU5T',
    id: '5Matrg5du62bXwer29cU5T',
    name: 'Benny The Butcher',
    type: 'artist',
    uri: 'spotify:artist:5Matrg5du62bXwer29cU5T'
  }
]
```

Why I did not use the id property is because the SpotifyExtractor SpotifySong interface only has these properties:
```
artists: {
    name: string;
    uri: string;
}[];
```

I will update the internal SpotifyTrack interface to add the uri property instead of replacing it:
```
artists: {
    id: string;
    name: string;
    uri: string;
}[];
```